pkg/occlient/occlient.go

```diff
@@ -444,11 +444,11 @@ func (c *Client) GetImageStreamsNames(namespace string) ([]string, error) {
 
 // NewAppS2I create new application using S2I
 // if gitUrl is ""  than it creates binary build otherwise uses gitUrl as buildSource
-func (c *Client) NewAppS2I(name string, builderImage string, gitUrl string, labels map[string]string, annotations map[string]string) (string, error) {
+func (c *Client) NewAppS2I(name string, builderImage string, gitUrl string, labels map[string]string, annotations map[string]string, stream bool) error {
```

Comment:
If you really want to have the `stream` flag then please add a comment what the stream argument is.